src/headless/tasks/inasafe_wrapper.py

```diff
@@ -266,9 +271,21 @@ def generate_report(impact_layer_uri, custom_report_template_uri=None):
 
     """
     output_metadata = read_iso19115_metadata(impact_layer_uri)
+    provenances = output_metadata.get('provenance_data', {})
+
+    hazard_layer = load_layer(
```

Comment:
So, we don't make the user able to set the layers for multiexposure analysis?

Reply:
this method is specific only for single IF, isn't it ?

Reply:
I was thinking to make it one method since they can differentiate it's single or multi exposure from the impact layer metadata.

Reply:
ok we'll do it in another PR